metis-authentication/metis-authentication-service/src/main/java/eu/europeana/metis/authentication/dao/ZohoApiFields.java

```diff
@@ -37,7 +37,8 @@
 	public static final String SORT_ORDER_DESC = "desc";
 	public static final String SORT_ORDER_ASC = "asc";
 	public static final String MODIFIED_TIME = "Modified Time";
-
+	public static final String ORGANIZATION_ROLE = "Organisation Role";
+	  
 	private static SimpleDateFormat formatter;
```

Comment:
I assume it is about the initialization of the formatter attribute. The purpose of this class is to keep the list of constants used for the invokation of the API and to avoid code dupplication. I'll move the initialization of the formatter from the get method to a static block. I hope that will solve the issue